examples/measure.js

```diff
@@ -0,0 +1,130 @@
+goog.provide('measure');
+
+goog.require('ngeo.DecorateInteraction');
+goog.require('ngeo.btngroupDirective');
+goog.require('ngeo.interaction.MeasureArea');
+goog.require('ngeo.interaction.MeasureAzimut');
+goog.require('ngeo.interaction.MeasureLength');
+goog.require('ngeo.mapDirective');
+goog.require('ol.FeatureOverlay');
+goog.require('ol.Map');
+goog.require('ol.Observable');
+goog.require('ol.Overlay');
+goog.require('ol.View');
+goog.require('ol.layer.Tile');
+goog.require('ol.source.OSM');
+
+
+/** @const **/
+var app = {};
+
+
+/** @type {!angular.Module} **/
+app.module = angular.module('app', ['ngeo']);
+
+
+/**
+ * App-specific directive wrapping the measure tools. The directive's
+ * controller has a property "map" including a reference to the OpenLayers
+ * map.
+ *
+ * @return {angular.Directive} The directive specs.
+ */
+app.measureToolsDirective = function() {
+  return {
+    restrict: 'A',
+    scope: {
+      'map': '=appMeasureToolsMap'
+    },
+    controller: 'AppMeasureToolsController',
+    controllerAs: 'measureCtrl',
```

Comment:
Minor but you can just use `ctrl` here. The directive uses an isolate scope so the property is very "local".